index.js

```diff
@@ -269,7 +269,13 @@ module.exports = (config) => {
 					};
 
 					const res = await testrail.addPlanEntry(config.plan.existingPlanId, data);
-					runId = res.runs[0].id;
+
+					if (config.runId) {
+						runId = config.runId;
+					} else {
+						runId = res.runs[0].id;
+					}
+
```

Comment:
could we do like this `runId = config.runId ? config.runId : res.runs[0].id;`

Reply:
@PeterNgTr sure.
 Fixed

Reply:
Perfect. Thanks!